plugin/pkg/shoot/validator/admission.go

```diff
@@ -47,6 +47,9 @@ import (
 const (
 	// PluginName is the name of this admission plugin.
 	PluginName = "ShootValidator"
+
+	useAsSeedAnnotation = "shoot.gardener.cloud/use-as-seed"
+	gardenNamespace     = "garden"
```

Comment:
Can you rather use https://github.com/gardener/gardener/blob/d2b1097a54781271f67c91c91f0f14ea6e969f94/pkg/apis/core/v1beta1/constants/types_constants.go#L230?